src/Symfony/Bundle/SecurityBundle/Security/Security.php

```diff
@@ -31,4 +35,58 @@ public function getFirewallConfig(Request $request): ?FirewallConfig
     {
         return $this->container->get('security.firewall.map')->getFirewallConfig($request);
     }
+
+    public function login(UserInterface $user, string $authenticatorName = null, string $firewallName = null): void
+    {
+        $request = $this->container->get('request_stack')->getCurrentRequest();
+
+        if (!class_exists(AuthenticatorInterface::class)) {
```

Comment:
unlikely to happen I think

Reply:
Hehe, indeed. So, should I remove the class_exists ?

Reply:
Please yes